### PR TITLE
Updated JupyterHub's REST API doc

### PR DESCRIPTION
**Pull request by Uzor13**
Updated the doc for JupyterHub's REST API doc for [issue 41](https://github.com/jupyterhub/outreachy/issues/41)

### CODE REVIEW

File: docs/source/getting-started/authenticators-users-basics.md [hunk 1]
```diff
@@ -25,10 +25,12 @@ If this configuration value is not set, then **all authenticated users will be a
 ```{note}
 As of JupyterHub 2.0, the full permissions of `admin_users`
 should not be required.
-Instead, you can assign [roles][] to users or groups
```
Comment (minrk) [inline docs/source/getting-started/authenticators-users-basics.md:28]: Why was this link removed?

File: docs/source/reference/rest.md [hunk 2]
```diff
@@ -1,47 +1,52 @@
-(rest-api)=
```
Comment (minrk) [inline docs/source/reference/rest.md:1]: This link target should stay

File: docs/source/reference/rest.md [hunk 3]
```diff
@@ -55,8 +60,12 @@ generating an API token is available from the JupyterHub user interface:
 
 ![Request API token page](../images/token-request.png)
 
+<p style="text-align: center">Figure 1.0: JupyterHub's Request API token page</p>
```
Comment (minrk) [inline docs/source/reference/rest.md:63]: inline html probably isn't the right way to add a caption. Let's try using the builtin [figure directive](https://jupyterbook.org/en/stable/content/figures.html#figures).

File: docs/source/reference/rest.md [hunk 4]
```diff
@@ -67,27 +76,28 @@ Prior to JupyterHub 2.0, there were two levels of permissions:
 where a token would always have full permissions to do whatever its owner could do.
 
 In JupyterHub 2.0,
-specific permissions are now defined as 'scopes',
+specific permissions are now defined as '**scopes**',
```
Comment (minrk) [inline docs/source/reference/rest.md:79]: 👍

File: docs/source/reference/rest.md [hunk 5]
```diff
@@ -136,7 +145,7 @@ Authorization header.
 ### Use requests
 
 Using the popular Python [requests](https://docs.python-requests.org)
-library, here's example code to make an API request for the users of a JupyterHub
+library, here's an example code to make an API request for the users of a JupyterHub
```
Comment (minrk) [inline docs/source/reference/rest.md:148]: I don't think the article here is correct. I would say "here is code" or "here is some code" not "here is a code"

**Review by minrk — COMMENTED** [review]
Thanks for your suggestions! I've made some comments to suggest changes to update your PR.

File: docs/source/getting-started/authenticators-users-basics.md (hunk 1 above)
Comment (Uzor13) [reply docs/source/getting-started/authenticators-users-basics.md:28]: It seems a commit for another doc authenticators-users-basics.md was added here, I've resolved them and it has been merged.

File: docs/source/reference/rest.md (hunk 2 above)
Comment (Uzor13) [reply docs/source/reference/rest.md:1]: It seems a commit for another doc authenticators-users-basics.md was added here, I've resolved them and it has been merged.

File: docs/source/getting-started/authenticators-users-basics.md [hunk 6]
```diff
@@ -101,19 +103,19 @@ system's UNIX users.
...
-- GitHub
-- GitLab
-- Globus
-- Google
-- MediaWiki
-- Okpy
-- OpenShift
-
-A generic implementation, which you can use for OAuth authentication
+- [Auth0](https://oauthenticator.readthedocs.io/en/latest/api/gen/oauthenticator.auth0.html#module-oauthenticator.auth0)
```
[the remark this answers is not in the document]
Comment (Uzor13) [reply docs/source/getting-started/authenticators-users-basics.md:106]: It seems a commit for another doc authenticators-users-basics.md was added here, I've resolved them and it has been merged.

File: docs/source/getting-started/authenticators-users-basics.md (hunk 1 above)
Comment (minrk) [reply docs/source/getting-started/authenticators-users-basics.md:28]: This removal and the links below are still present. Maybe your fixes didn't get pushed? These changes can be reverted, the rest is great!

Comment (Uzor13) [reply docs/source/getting-started/authenticators-users-basics.md:28]: can i close this PR and open another? seems I made a mistake on my local and I don't want to push that

Comment (minrk) [reply docs/source/getting-started/authenticators-users-basics.md:28]: Yes, if that's the easiest way for you, that's fine.

You can also try rebasing by using `git reset origin/main` and then only committing the changes you want to keep.

There's no difference between opening a new PR and force-pushing to the branch used by this PR. If you do open a new PR, please close this one with a link to a new one.

**Comment by Uzor13** [comment]
This is the link to the new PR: https://github.com/jupyterhub/jupyterhub/pull/4157

**Pull request closed without merging**